### ISSUE TITLE
a small question in DAY1 class 

### ISSUE THREAD

**Issue by wangzy20** [Scavetta/conf_tensorflow_training_day1#2]
Hi,

I am learning your great tutor of keras in R but I can't understend some point  in  the 1_3_DL_Basics_Multi-class_Classification.Rmd though it is really a detailed tutor I have ever seen.

`network <- keras_model_sequential() %>% 
  layer_dense(units = 2^6, activation = "relu", input_shape = 13) %>% 
  layer_dense(units = 2^6, activation = "relu") %>% 
  layer_dense(units = 5, activation = "softmax")
  
network %>% compile(
  optimizer = "rmsprop",
  loss = "sparse_categorical_crossentropy",
  metrics = "accuracy"
)

history<- network %>% fit(
  train_data,
  train_targets,
  epochs = 50,
  batch_size = 16,
  verbose = FALSE
)`

In these code, we seems haven't saved the  parameter to our network to our 'network' model 
How can use the optimize result to the model?

Thanks
